packages/headless/src/modules/order/graphql/order.admin.model.ts

```diff
@@ -0,0 +1,194 @@
+import { Field, InputType, Int, ObjectType, registerEnumType } from "@nestjs/graphql";
+import { 
+    GetAllOrderQuery, 
+    OrderIncompleteStat, 
+    OrderStat, 
+    OrderStatusEnumModel, 
+    PaymentStatusEnumModel, 
+    ShippingStatusEnumModel, 
+    OrderStatusModel,
+    ChangeStatusModel
+} from "models";
+
+import { OrderResponse } from "./order.customer.model";
+
+@ObjectType({description: 'Order Status Enum Model'})
```

Comment:
Instead of object type use enum type
https://docs.nestjs.com/graphql/unions-and-enums#code-first-1